models/src/main/scala/coop/rchain/models/rholangn/parmanager/Serialization.scala

```diff
@@ -0,0 +1,581 @@
+package coop.rchain.models.rholangn.parmanager
+
+import com.google.protobuf.{CodedInputStream, CodedOutputStream}
+import coop.rchain.models.rholangn._
+import coop.rchain.models.rholangn.parmanager.Constants._
+
+import java.io.{InputStream, OutputStream}
+
+private[parmanager] object Serialization {
```

Comment:
I guess this is `ProtobufSelialization`, serialization specific to protobuf?

Reply:
hm, I used protobuf for only low level serialization, but I'm not sure that top serialization similar as protobuff. I think it's some kind of hybrid of protobuff and self-written stuff.

Reply:
Maybe this an be a case class accepting a trait that provides (de)serialization of basic objects.